x-pack/plugins/security_solution/public/flyout/document_details/shared/hooks/use_show_related_alerts_by_ancestry.ts

```diff
@@ -71,24 +59,14 @@ export const useShowRelatedAlertsByAncestry = ({
 
   const ancestorId = getField(getFieldsData(ANCESTOR_ID)) ?? '';
   const documentId = isPreview ? ancestorId : eventId;
-  // can't use getFieldsData here as the kibana.alert.rule.parameters is different and can be nested
-  const originalDocumentIndex = useMemo(
```

Comment:
here also I just want to make sure we're ok with removing the `originalDocumentIndex` check. I'll reach out on Slack so we can discuss

Reply:
discussed it on Zoom, this new behavior is what we were already following for analyzer so we're ok!